### ISSUE TITLE
Calendar tab throwing "Error getting events {}" 

### ISSUE THREAD

**Issue by samarthsinghal1** [microsoftgraph/msgraph-training-react-native#20]
Running the app to see the JSON dump of the calendar events and getting "Error getting events {}" 
![Screenshot 2021-04-07 at 6 40 33 AM](https://user-images.githubusercontent.com/33370925/113796135-294b6080-976c-11eb-8514-3bb99c7a6429.png) and on clicking "OK" it keeps on throwing the loading screen.

https://user-images.githubusercontent.com/33370925/113796254-70d1ec80-976c-11eb-93fa-97591f0997ac.mov

**Comment by jasonjoh** [comment]
What's the error?

**Comment by samarthsinghal1** [comment]
Was having library import error with findOneIana method in the CalendarScreen.tsx, changing it to findIana and setting `const ianaTimeZone = findIana(tz)[0];` fixed it.

**Comment by jasonjoh** [comment]
Hmm. What version of windows-iana do you have installed?

**Comment by JBulteau** [comment]
Hello, I apologize if I should have opened another issue for this.
I got the same problem and figured out that the function findOneIana no longer exists in windows-iana. It has been removed in version 5.0.0 of the module, released on January 29th, 2021 (see attached). Using findIana and getting the proper value in the array, as the author has suggested, works indeed.
![image](https://user-images.githubusercontent.com/33154521/115559446-04023a80-a2b4-11eb-86ba-cc53034583f4.png)
